docs/pages/server-access/guides/azure-discovery.mdx

```diff
@@ -0,0 +1,271 @@
+---
+title: Configure Teleport to Automatically Enroll Azure Virtual Machines (Preview)
+description: How to configure Teleport to automatically enroll Azure Virtual Machines.
+---
+
+The Teleport Discovery Service can connect to Azure and automatically
+discover and enroll virtual machines matching configured labels. It will then
+execute an install script on these discovered instances that will install
+Teleport, start it and join the cluster.
+
+In this scenario, Teleport Discovery Service uses an Azure invite token with a
+long time-to-live (TTL), so that new instances can be discovered and added to
+the Teleport cluster for the lifetime of the token.
+
+## Prerequisites
+
+(!docs/pages/includes/edition-prereqs-tabs.mdx!)
+
+- Azure subscription with virtual machines and permissions to create and attach
+managed identities.
+- Azure virtual machines running Ubuntu/Debian/RHEL if making use of the
+default Teleport install script. (For other Linux distributions, you can
+install Teleport manually.)
+
+(!docs/pages/includes/tctl.mdx!)
+
+## Step 1/5. Create an Azure invite token
+
+When discovering Azure Virtual Machines, Teleport makes use of Azure invite tokens for
+authenticating joining Nodes.
```

Comment:
```suggestion
authenticating joining SSH Service instances.
```
We're moving away from the "Node" language (see the core concepts page: https://goteleport.com/docs/core-concepts/).